src/atomate2/common/jobs/phonons.py

```diff
@@ -282,84 +295,49 @@ def run_phonon_displacements(
         "dirs": [],
     }
 
-    for i, displacement in enumerate(displacements):
-        if prev_vasp_dir is not None:
-            phonon_job = phonon_maker.make(displacement, prev_vasp_dir=prev_vasp_dir)
+    if socket:
+        if prev_dir is not None:
+            phonon_job = phonon_maker.make(displacements, prev_dir=prev_dir)
```

Comment:
@utf I am wondering if we should in general switch to prev_dir.

Reply:
I agree that will be a breaking change and additional work, but as more and more codes are coming and we work on "common" workflows, it is probably something we have to do sooner or later. We might also need to discuss how to connect jobs from different codes in the best way.

Reply:
When I made the change I was not sure what the overall plan was for a generalization like this. I am happy to help in the discussions about connecting jobs from different codes. 

Let me know if you want me to make a change here in the VASP jobs for `prev_dir` vs `prev_vasp_dir`

Reply:
See below for the solution